documentation/onboarding.md

```diff
@@ -0,0 +1,169 @@
+
+![Image showing the installer credentials dialog highlighting the Install Software button](./images/onboarding/installer-credentials.png)
+
+Allow a few minutes for the installation to complete. A dialog appears confirming the installation was successful. Select **[!UICONTROL Close]**.
+
+![Image showing the installer dialog on the Summary tab confirming installation was successful highlighting the Close button](./images/onboarding/installation-successful.png)
+
+The Agenda app is now available in the **Launchpad**.
+
+![Image showing the Agenda app in the Launchpad](./images/onboarding/launchpad.png)
+
+## Create an account
+
+An Agenda account gives you access to many Agenda products. With an Agenda account you can do the following:
+
+* Manage meetings
+* Manage teams
+* Integrate external calendars
+* Manage multiple calendars in one space
+
+To create a new account:
+
+Navigate to the desktop application and select to open. The sign in screen appears. Select **[!UICONTROL Create an account]** on the right side of the screen.
+
+![Image showing the Agenda desktop app Sign-In screen highlighting the Create and account button](./images/onboarding/create-an-account.png)
+
+The **[!UICONTROL Create an account]** screen appears. Enter your name, email address, password, and confirm your password. Select **[!UICONTROL Create account]** to complete the account creation process.
+
+![Image showing the Create an acocunt screen highlighting required fields and the Createa account button](./images/onboarding/complete-create-account.png)
+
+Confirmation of successful account creation appears. Select **[UICONTROL Close]** and you are returned to the **Sign In** screen.
+
+![Image showing the successful account creation dialog highlighting the Close button](./images/onboarding/account-creation-successful.png)
+
+## Integrate an external calendar
+
+In Agenda you can integrate an external calendar allowing you to import and sync events and manage these from a single space.
+
+To integrate an external calendar:
+
+Log into the Agenda application and select the **[!UICONTROL Sources]** tab from the top navigation, then select **[!UICONTROL Add Calendar sources]** from the left navigation.
+
+![Image highlighting the Sources tab and Calendar sources selected](./images/onboarding/calendar-sources.png)
+
+A list of available sources is displayed. Find the service provider for your account and select connect.
+
+![Image highlighting the calendar source selection](./images/onboarding/select-source.png)
+
+A login dialog appears. Enter your email address and password for the account and then select **[UICONTROL Connect]**.
+
+![Image showing the calendar source login dialog box higlighting the Connect button](./images/onboarding/calendar-source-login.png)
+
+A list of calendars associated with that account appear. Select the calendars that you would like to import by selecting the check box beside the calendars name, the select **[!UICONTROL Import calendars]**.
```

Comment:
I think it should be "...associated with _the_ account appear."

Second sentence: "Select the check box beside the name of each calendar that you would like to import, then select **Import calendars**."